docs/pages/accounts/two-factor.mdx

```diff
@@ -24,15 +24,15 @@ Expo accepts any authenticator app that supports Time-based One-time Passwords (
 
 Expo will provide a QR code to scan with your authenticator app during setup. The app will provide a confirmation code to enter on Expo. Enter the code to finish activating 2FA via your authenticator app.
 
-### SMS Messages
+### SMS Messages (deprecated)
```

Comment:
```suggestion
### SMS Messages
```

Once we change below callout to use warning, explicitly mentioning "deprecated" here doesn't make sense. The callout is enough to signify that this feature is deprecated.